sdks/python/setup.py

```diff
@@ -168,7 +168,7 @@ def get_version():
     'mock>=1.0.1,<3.0.0',
     'nose>=1.3.7',
     'nose_xunitmp>=0.4.1',
-    'pandas>=1.0,<1.3.0',
+    'pandas==1.3.0rc1',
```

Comment:
```suggestion
    'pandas>=1.0,>1.4.0',
```

Now that 1.3.0 is out this should be changed.